modules/osd-aws-privatelink-about.adoc

```diff
@@ -0,0 +1,7 @@
+[id="osd-aws-privatelink-about.adoc_{context}"]
+= Understanding PrivateLink
+
+A {product-title} cluster can be created without any requirements on public subnets or public internet access. In this configuration, Red Hat uses AWS PrivateLink (PrivateLink) to manage and monitor a cluster in order to avoid any public ingress or egress network traffic.
```

Comment:
`A {product-title} cluster can be created without any requirements on public subnets or public internet access.`
Should it be public here?

Reply:
Hi @xueli181114, - Just a clarification,  are you suggesting to remove public from in front of internet access (It seems a bit redundant), or something else?

Reply:
@xueli181114 @kmccarron-rh - PrivateLink clusters actually need public egress in order to install OCP components. Can we reword this to just public ingress endpoints?

Reply:
@dustman9000 It's a little nuanced though, right? It's _technically_ public egress, but it's going through the customer's private network. I just don't want the customer to think that we're bypassing something

Reply:
@wgordon17  Not sure what the takeaway action is here - Can you clarify what change, if any, needs to be made here?

Reply:
s/ A {product-title} cluster can be created without any requirements on public subnets or public internet access. In this configuration, Red Hat uses AWS PrivateLink (PrivateLink) to manage and monitor a cluster in order to avoid any public ingress or egress network traffic. 
/ A {product-title} cluster can be created using AWS PrivateLink (PrivateLink) to allow Red Hat to manage and monitor a cluster through a customer's private network. Although Red Hat accesses the private cluster through public ingress, the cluster is secured by AWS PrivateLink.

@wgordon17 @dustman9000 @kmccarron-rh WDYT of my suggestion? If needed, the last sentence could be deleted entirely. But I think the first two sentences may be accurate enough.

Reply:
I had actually made a suggestion down further (https://github.com/openshift/openshift-docs/pull/32552#discussion_r646879608), GH didn't seem to tie it all together too well :(

Reply:
Ah - got it. Thanks